test/dkms_per_module_config_test-1.0/dkms.conf

```diff
@@ -0,0 +1,26 @@
+PACKAGE_NAME="dkms_per_module_config_test"
+PACKAGE_VERSION="1.0"
+
+# Multiple modules with different per-module BUILD_EXCLUSIVE configurations
+BUILT_MODULE_NAME[0]="dkms_per_module_config_test_mod1"
+BUILT_MODULE_NAME[1]="dkms_per_module_config_test_mod2"
+BUILT_MODULE_NAME[2]="dkms_per_module_config_test_mod3"
+
+DEST_MODULE_LOCATION[0]="/updates/dkms"
+DEST_MODULE_LOCATION[1]="/updates/dkms"
+DEST_MODULE_LOCATION[2]="/updates/dkms"
+
+# Per-module kernel version exclusions
```

Comment:
I believe these conditions are inverted? Exclusive would mean only in "this particular" case, right? If not this usage is inconsistent with the test `test/dkms_per_module_config_exclusive_test-1.0/dkms.conf`, which matched my understanding as requiring a matching (exclusive to) config.

For the cases below:
- 0: Only build when kernel version is 99.x
- 1: Build when kernel version is anything
- 2: This is the same as 2?

Reply:
I changed the text completely so it's more clear.